src/main.lua

```diff
@@ -138,6 +138,25 @@ local function addRecordResultFields(ioEvent, result)
 	end
 end
 
+local function addAuctionResultFields(ioEvent, result)
+	ioEvent:addFieldsIfExist(result, {
+		"bidAmount",
+		"bidder",
+		"rewardForInitiator",
+		"rewardForProtocol",
+		-- TODO: we could return some computed values of the bid relative to start and end price
```

Comment:
Is it hard to get the start and end price values of the auction? I think it would be very valuable data to have here.

Reply:
we compute it based on the bid terms (years, and type) - we need to bubble it up from other places

Reply:
added here - [5931635](https://github.com/ar-io/ar-io-network-process/pull/154/commits/593163517498d51631ceb6c39cd5f54592398589)